test/acceptance/features/bindAppToService.feature

```diff
@@ -216,4 +216,31 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
-
+    Scenario: Bind an imported Node.js application to Etcd database
+        Given Etcd operator running
+        * Etcd cluster "etcd-cluster-example" is running
+        * Imported Nodejs application "nodejs-rest-http-crud-etcd" is running
```

Comment:
@shvasude This nodes application will not work with etcd database since it uses postgresql database. Please follow https://github.com/redhat-developer/service-binding-operator/tree/master/examples/nodejs_etcd_operator

Reply:
@akashshinde I tried with this nodejs and received this `apiVersion: apps.openshift.io/v1alpha1
kind: ServiceBindingRequest
metadata:
  name: binding-request-etcd
  namespace: test-namespace-b2c50eab
spec:
  applicationSelector:
    group: apps
    version: v1
    resource: deployments
    resourceRef: node-todo-git-app
  backingServiceSelector:
    group: etcd.database.coreos.com
    version: v1beta2
    kind: EtcdCluster
    resourceRef: etcd-cluster-example
  detectBindingResources: true`

Hence used another nodejs application and works everything except validating api end point.

Reply:
so how this work if validating endpoint does not work?

Reply:
I thought it works if injectionReady and collectionReady are True and nodejs app fruit app results True for both the status

Reply:
This commit fixes the use of nodejs todo app instead of nodejs fruit app. 
https://github.com/shvasude/service-binding-operator/commit/e0d93ef37eeb73a61859e5e6220a063a7d4ea67c